contracts/conduit/Conduit.sol

```diff
@@ -167,14 +207,19 @@ contract Conduit is ConduitInterface, TokenTransferrer {
     }
 
     /**
-     * @dev Internal function to transfer a given ERC20/721/1155 item.
+     * @dev Internal function to transfer a given ERC20/721/1155 item. Note that
+     *      channels are expected to implement checks against transferring any
+     *      zero-amount items if that constraint is desired.
      *
      * @param item The ERC20/721/1155 item to transfer.
      */
     function _transfer(ConduitTransfer calldata item) internal {
-        // If the item type indicates Ether or a native token...
+        // Determine the transfer method based on the respective item type.
         if (item.itemType == ConduitItemType.ERC20) {
-            // Transfer ERC20 token.
+            // Transfer ERC20 token. Note that item.identifier is ignored and
```

Comment:
Is there a particular reason for this?

Reply:
Avoiding a redundant check mainly, basically letting channels make the call as to whether they want to do this safety check or not

Reply:
> Avoiding a redundant check mainly,

Where is this checked before hand? The conduit can be used independently, can't it be?